src/routes/v1/capp_test.go

```diff
@@ -84,7 +84,7 @@ func TestPatchCapp(t *testing.T) {
 		Status: cappv1.CappStatus{},
 	}
 	body, _ := json.Marshal(cappRequest)
-	request, _ := http.NewRequest("PATCH", "/v1/namespaces/test-namespace/capps/test-capp", bytes.NewBuffer(body))
+	request, _ := http.NewRequest("PUT", "/v1/namespaces/test-namespace/capps/test-capp", bytes.NewBuffer(body))
```

Comment:
We should use `http.MethodPut` to minimize the amount of hardcoded strings in the code

Reply:
You're right, I can open an issue for it and handle it in a new PR, because I'd like to keep this PR under single purpose to change only from `PATCH` to `PUT`.